etc/module.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="AvS_ScopeHint" setup_version="1.0.0">
```

Comment:
Dropping this one will make the module no longer installable on Magento 2.0.x, 2.1.x and 2.2.x, so this should be reflected in the composer constraints I think.

Reply:
@hostep Good catch. Do you happen to know which version of `magento_framework` is responsible for that change? I couldn't find out yet. Currently we have:
```
    "require": {
        "magento/framework": "~100.0|~101.0|~102.0|~103.0",
        "magento/module-config": "~100.0|~101.0"
    },
``` 
Otherwise I'll just revert that patch :-/

Reply:
To me it's the same if you want to keep supporting Magento 2.2.x and lower or not :)

As far as the versions go, the `setup_version`:
- [was required](https://github.com/magento/magento2/blob/2.2.11/lib/internal/Magento/Framework/Module/etc/module.xsd#L46) in Magento 2.2.11
- [is optional](https://github.com/magento/magento2/blob/2.3.0/lib/internal/Magento/Framework/Module/etc/module.xsd#L46) since Magento 2.3.0

Framework versions:
- Magento 2.2.11 uses [framework version 101.0.11](https://github.com/magento/magento2/blob/2.2.11/lib/internal/Magento/Framework/composer.json#L5)
- Magento 2.3.0 uses [framework version 102.0.0](https://github.com/magento/magento2/blob/2.3.0/lib/internal/Magento/Framework/composer.json#L59)

Reply:
Thank you! I declared compatiblity to magento/framwork 102 and 103 only. I don't think that Magento 2.2 and lower is used significantly any more.